index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Yasir's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="app-bar">
+        <div class="logo-placeholder">
+            <div class="place valign-text-middle">Welcome</div>
+        </div>
+        <div class="normal-button">
+            <div class="icons-icon">
+                <img class="union-2" src="img/union-2.svg" alt="menu icon">
+            </div>
+        </div>
+    </div>
+    <div class="headline">
+        <h1 class="primary">
+            <span class="inter-bold-blue-zodiac-40px">Hey there. I'm Yasir<br /></span>
+            <span class="span1">I'm Software Developer</span>
+        </h1>
+        <div class="supporting-text">
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hesitate to contact me.
+        </div>
+        <img class="social-media-icons" src="img/socialmediaicons.svg" alt="social media icons">
+        <div class="action-1">
+            <div class="normal-button-secondary">
+                <div class="icons-icon">
+                    <img class="union" src="img/Action.png" alt="dropdown icon">
```

Comment:
- [ ] For this button, Please use a link `a` tag for a semantic reason and the `a` tag will let you connect to other sections of the website, using href. :smile: